### PR TITLE
New plugin version

### PR DESCRIPTION
**Pull request by mateuszpy**
Old plugin version (1.4.0) isn't working with new PrestaShop versions so here is updated one with fixes necessary to run on PrestaShop 1.6.1.0. Plugin has been checked using validator at http://validator.prestashop.com

### CODE REVIEW

File: dotpay.php [hunk 1]
```diff
@@ -82,6 +82,8 @@ public function install()
     {
 	
         Configuration::updateValue('DP_TEST', true);
+        Configuration::updateValue('DP_CHK', false);
+        Configuration::updateValue('DP_SSL', false);
```
Comment (Quetzacoalt91) [inline dotpay.php]: Maybe you could add these configuration keys in an upgrade file, or these configuration values won't exist if the module is already installed..

**Comment by Quetzacoalt91** [comment]
Hi @mateuszpy,

Why did you duplicate payment_return.tpl in /front and /hook folders in this pull-request ?

**Comment by mateuszpy** [comment]
We addressed both of your comments and made some small changes in the module.
Duplicate was made for compatibility some time ago but it's not needed anymore.

**Comment by Quetzacoalt91** [comment]
Hi @mateuszpy,

Thanks, your PR will be merged on Monday.

**Comment by mateuszpy** [comment]
Thanks for the information @Quetzacoalt91 
Does it mean updated plugin is going to appear on the list of PrestaShop modules on Monday?

**Comment by Quetzacoalt91** [comment]
Hi @mateuszpy,

Thanks, your PR has just been merged and the changes will be available from tomorrow.

Best regards

**Comment by mateuszpy** [comment]
Thank you @Quetzacoalt91

**Pull request merged**